rules/library.bzl

```diff
@@ -827,12 +824,18 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             generated_header_name = generated_swift_header_name,
             generates_header = True,
             srcs = swift_sources,
-            copts = copts_by_build_setting.swift_copts + swift_copts + additional_swift_copts,
-            deps = deps + private_deps + lib_names,
+            # Note: by default it used a vfs but not the entire virtual framwork
+            # feature.
+            copts = select({
+                "@build_bazel_rules_ios//:virtualize_frameworks": framework_vfs_swift_copts,
+                "//conditions:default": framework_vfs_swift_copts if enable_framework_vfs else [],
+            }) + copts_by_build_setting.swift_copts + swift_copts + additional_swift_copts,
+            deps = deps + private_deps + lib_names + select({
+                "@build_bazel_rules_ios//:virtualize_frameworks": [framework_vfs_overlay_name_swift],
```

Comment:
to make sure I understand correctly: in either situation we need to include vfs info: this feature is enabled via a condition (aka transitive) OR enable_framework_vfs is set to TRUE. Am I correct here?

Reply:
Yep it uses a `select` statement to switch on `"@build_bazel_rules_ios//:virtualize_frameworks"`, and add other copts and deps. Longer term, I suspect it would be the default